src/main/java/de/hysky/skyblocker/skyblock/waypoint/WaypointsListWidget.java

```diff
@@ -33,7 +31,7 @@ public WaypointsListWidget(MinecraftClient client, AbstractWaypointsScreen<?> sc
 
     @Override
     public int getRowWidth() {
-        return super.getRowWidth() + 100;
+        return super.getRowWidth() + 110;
```

Comment:
Since you are adding an element with width 11 and the gap between elements is 5 (I think), you should add 16 (116). Then you should move the elements that depend on x to the right by 16, and move all the elements that depend on x + width / 2 by 8, and don’t move the elements that depend on x + width. Right now it’s a bit all over the place. Thanks.